### DISCUSSION TITLE
Integration with PowerToys

### DISCUSSION THREAD

**Discussion by HeekangPark** [ShankarBUS/ModernFlyouts#1097] [Ideas]
Hi, thank you for making an awesome app!

Have you ever heard of [PowerToys](https://github.com/microsoft/PowerToys)? Is there any plan to integrate ModernFlyouts with PowerToys?

**Comment by Samuel12321** [comment]
hi, thanks.
There are no current plans to integrate with powertoys. It has been discussed before but nothing happened.
